i18n/zh-cn/docusaurus-plugin-content-docs/current/api/connect.md

```diff
@@ -68,15 +63,19 @@ const gearApi = await GearApi.create({ customTypes: { ...yourCustomTypesExample
 async function connect() {
   const gearApi = await GearApi.create();
 
-  const [chain, nodeName, nodeVersion] = await Promise.all([
+  const [chain, nodeName, version] = await Promise.all([
```

Comment:
```js
  const [chain, nodeName, nodeVersion] = await Promise.all([
    gearApi.chain(),
    gearApi.nodeName(),
    gearApi.nodeVersion(),
  ]);
```
The previous code worked well， you can try this code https://github.com/GearFans/example/blob/main/src/connect.ts

Reply:
> ```js
>   const [chain, nodeName, nodeVersion] = await Promise.all([
>     gearApi.chain(),
>     gearApi.nodeName(),
>     gearApi.nodeVersion(),
>   ]);
> ```
> 
> The previous code worked well， you can try this code https://github.com/GearFans/example/blob/main/src/connect.ts

Thanks for the comments. Pls help to try the EN version. CN version should align with the EN version.